mainwindow.cpp

```diff
@@ -12,13 +12,14 @@ using jbrush::Canvas;
 
 MainWindow::MainWindow(QWidget *parent) :
     QMainWindow(parent),
-    ui(new Ui::MainWindow)
+    ui(new Ui::MainWindow),
+    currentFile(""),
```

Comment:
Should be unnecessary. ``QString`` should create a empty string upon the default ctor.